modules/ROOT/pages/sysconfig-network-configuration.adoc

```diff
@@ -99,12 +99,15 @@ See https://coreos.github.io/afterburn/usage/initrd-network-cmdline/[the Afterbu
 
 WARNING: If you need networking to grab your Ignition config and your environment requires more complex networking than the default of DHCP to grab the Ignition config, then you'll need to use another method other than Ignition to configure the network.
 
-Networking configuration can be performed by writing out files described in an Ignition config. These are https://networkmanager.dev/docs/api/latest/nm-settings-keyfile.html[NetworkManager keyfiles] that are written to `/etc/NetworkManager/system-connections/` that tell NetworkManager what to do.
+There are to main methods for configuring networking with ignition 
```

Comment:
```suggestion
There are two main methods for configuring networking by delivering text configuration files via Ignition:
```

Also Let's add a blank line after this line so the bullets get broken out properly by antora/asciidoctor